connect/soap-http-client/src/test/java/org/operaton/connect/soap/httpclient/SoapHttpConnectorSystemPropertiesTest.java

```diff
@@ -60,7 +51,7 @@ public class SoapHttpConnectorSystemPropertiesTest {
   @Before
   public void setUp() {
     updatedSystemProperties = new HashSet<String>();
-    wireMockRule.stubFor(get(urlEqualTo("/")).willReturn(aResponse().withStatus(200)));
+    wireMockRule.stubFor(post(urlEqualTo("/")).willReturn(aResponse().withStatus(200)));
```

Comment:
**Review Hint**: The test failed because SOAP requests are sent via POST, but stub was for GET. I don't know why it worked before the upgrade, feels like a bug that was not detected.